### PR TITLE
use pure cmake to build and install the package (remove ament)

### PR DESCRIPTION
**Pull request by MaximilienNaveau**
[//]: # "Thanks for your contribution.  To make life of the reviewers easier,"
[//]: # "please give this pull request a meaningful title and provide the"
[//]: # "requested information below."

## Description

[//]: # "Description of what you did.  If it is more complex, consider to add a"
[//]: # "'Summary' section."

Remove the dependency toward AMENT and use the mpi_cmake_module instead.

## How I Tested

[//]: # "Explain how you tested your changes"

I build using colcon and ran the unit-tests

## I fulfilled the following requirements

[//]: # "Please make sure you followed these steps before requesting a review."
[//]: # "Check the boxes in the list below, when done."

- [x] All new code is formatted according to our style guide (for C++ run clang-format, for Python, run flake8 and fix all warnings).
- [x] All new functions/classes are documented and existing documentation is updated according to changes.
- [x] No commented code from testing/debugging is kept (unless there is a good reason to keep it).

### CODE REVIEW

**Review by ahmadgazar — APPROVED** [review]
good job, tested it and it works for me now in Release.

File: Config.cmake.in [hunk 1]
```diff
@@ -0,0 +1,20 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
+
+include(CMakeFindDependencyMacro)
+
+# we do not add the other dependencies because these are header files lib
+if(${CMAKE_VERSION} VERSION_LESS "3.15.0")
```
Comment (luator) [inline Config.cmake.in:8]: Same question as on other packages: Is this distinction needed?

**Review by luator — APPROVED** [review]
Looks good to me.  Only question is if the version distinction in Config.cmake.in is actually needed for minimum version 3.10.2.

File: Config.cmake.in (hunk 1 above)
Comment (MaximilienNaveau) [reply Config.cmake.in:8]: Yes otherwise the dependencies are not seen by child packages.

**Comment by MaximilienNaveau** [comment]
> Looks good to me. Only question is if the version distinction in Config.cmake.in is actually needed for minimum version 3.10.2.

Yes unfortunately.
This is to be removed of course when the ubuntu version will climb up and that the default cmake version follows.

File: Config.cmake.in (hunk 1 above)
Comment (luator) [reply Config.cmake.in:8]: I see.  All good then :+1:

**Pull request merged**